setup.cfg

```
@@ -94,3 +94,7 @@ warn_unreachable = true
 [codespell]
 ignore-words-list = coo,nd
 skip = docs/source/changelog.rst
+
+[options.entry_points]
+dask.backends =
```

I think we're missing the `array` portion here 

```suggestion
dask.array.backends =
```